build/run-me.sh

```diff
@@ -377,32 +397,41 @@ if [ -z "${SSH_AUTH_SOCK+false}" ]; then
 fi
 
 # Build the docker build environment
-docker build -f "$execdir/$dockerfile" -t "$imagename" "$execdir"
+docker build -f "$dockerfile_path" -t "$imagename" "$execdir"
 
 if [ -n "${external_manifest:-}" ]; then
   name="$(basename "$external_manifest")"
   cp "$external_manifest" "$builddir/$name"
   set -- "--external-manifest=$builddir/$name" "$@"
 fi
 
-# The ${:+} expansion of download upsets shellcheck, but we do not
-# want that instance quoted because that would inject an empty
-# argument when download is not defined.
-# shellcheck disable=SC2086
-docker run --rm -i $flag_tty \
-       --name "$default_containername" \
-       -e LOCAL_UID="$(id -u)" -e LOCAL_GID="$(id -g)" \
-       -e SSH_AUTH_SOCK="$SSH_AUTH_SOCK" \
-       ${downloaddir:+-v "$downloaddir":"$downloaddir"} \
-       ${outputdir:+-v "$outputdir":"$outputdir"} \
-       -v "$(dirname "$SSH_AUTH_SOCK"):$(dirname "$SSH_AUTH_SOCK")" \
-       -v "$builddir":"$builddir" \
-       ${privileged_arg} \
-       "$imagename" \
-       ./${build_script} --builddir "$builddir" \
-         ${build_args:-} \
-         ${downloaddir:+--downloaddir "$downloaddir"} \
-         ${outputdir:+--outputdir "$outputdir"} \
-         --parent-command-line "$command_line" \
-         ${mbl_tools_version:+--mbl-tools-version "$mbl_tools_version"} \
-         "$@"
+
+if [ "$project" == "dockcross" ]; then
+  cross_build_script="$imagename"
```

Comment:
Are you going to pass a custom --image-name option like `dockcross/linux-armv7` or `dockcross/linux-arm64`?